charts/cloudzero-agent/values.yaml

```diff
@@ -182,7 +182,7 @@ initCertJob:
     serviceAccountName: ""
     clusterRoleName: ""
     clusterRoleBindingName: ""
-  ttlSecondsAfterFinished: 180
+  ttlSecondsAfterFinished: 5
```

Comment:
I'm bumping this down; we probably want to run this in upgrades that happen one after another